spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/iceberg/IcebergPropertiesConstants.java

```diff
@@ -0,0 +1,40 @@
+/*
+ *  Copyright 2024 Datastrato Pvt Ltd.
+ *  This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.spark.connector.iceberg;
+
+import com.datastrato.gravitino.catalog.lakehouse.iceberg.IcebergCatalogPropertiesMetadata;
+import com.google.common.annotations.VisibleForTesting;
+
+public class IcebergPropertiesConstants {
+
+  @VisibleForTesting
+  public static final String GRAVITINO_ICEBERG_CATALOG_BACKEND =
+      IcebergCatalogPropertiesMetadata.CATALOG_BACKEND_NAME;
+
+  @VisibleForTesting
+  public static final String GRAVITINO_ICEBERG_CATALOG_WAREHOUSE =
+      IcebergCatalogPropertiesMetadata.WAREHOUSE;
+
+  @VisibleForTesting
+  public static final String GRAVITINO_ICEBERG_CATALOG_URI = IcebergCatalogPropertiesMetadata.URI;
+
+  public static final String GRAVITINO_ICEBERG_JDBC_USER =
```

Comment:
what's the difference of `GRAVITINO_ICEBERG_CATALOG_JDBC_USER` and `GRAVITINO_ICEBERG_JDBC_USER`? seems confusing to understand.

Reply:
suggest to remove `CATALOG` for `GRAVITINO_ICEBERG_CATALOG_JDBC_USER` `GRAVITINO_ICEBERG_CATALOG_JDBC_PASSWORD` `GRAVITINO_ICEBERG_CATALOG_JDBC_DRIVER`

Reply:
> what's the difference of `GRAVITINO_ICEBERG_CATALOG_JDBC_USER` and `GRAVITINO_ICEBERG_JDBC_USER`? seems confusing to understand.

In order to distinguish `jdbc-user` and `jdbc.user` in `IcebergCatalogPropertiesMetadata`
```
  public static final String GRAVITINO_JDBC_USER = "jdbc-user";
  public static final String ICEBERG_JDBC_USER = "jdbc.user";

  public static final String GRAVITINO_JDBC_PASSWORD = "jdbc-password";
  public static final String ICEBERG_JDBC_PASSWORD = "jdbc.password";
```

Reply:
> suggest to remove `CATALOG` for `GRAVITINO_ICEBERG_CATALOG_JDBC_USER` `GRAVITINO_ICEBERG_CATALOG_JDBC_PASSWORD` `GRAVITINO_ICEBERG_CATALOG_JDBC_DRIVER`

change `GRAVITINO_ICEBERG_CATALOG_JDBC_USER` `GRAVITINO_ICEBERG_CATALOG_JDBC_PASSWORD` `GRAVITINO_ICEBERG_CATALOG_JDBC_DRIVER` to `GRAVITINO_ICEBERG_JDBC_USER` `GRAVITINO_ICEBERG_JDBC_PASSWORD` `GRAVITINO_ICEBERG_JDBC_DRIVER`?

and change `GRAVITINO_ICEBERG_JDBC_USER` `GRAVITINO_ICEBERG_JDBC_PASSWORD` to `GRAVITINO_JDBC_USER` `GRAVITINO_JDBC_PASSWORD` ?

WDYT?